src/libaktualizr/primary/sotauptaneclient.cc

```diff
@@ -76,54 +76,52 @@ data::InstallationResult SotaUptaneClient::PackageInstall(const Uptane::Target &
 }
 
 void SotaUptaneClient::finalizeAfterReboot() {
-  if (!package_manager_->rebootDetected()) {
-    // nothing to do
+  // TODO: consider bringing checkAndUpdatePendingSecondaries and the following functionality
+  // to the common denominator
+  if (!hasPendingUpdates()) {
+    LOG_INFO << "No any pending update, continue with initialization";
     return;
   }
 
-  LOG_INFO << "Device has been rebooted after an update";
+  LOG_INFO << "Check if there is any pending update for Primary ECU to be applied.";
```

Comment:
This text reads like a suggestion to the user, whereas it's a description of the current operation.

```suggestion
  LOG_INFO << "Checking for pending update for primary ECU to apply";
```

Reply:
ok, will update accordingly